src/examples/forms/input-group/Size.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { mediaQuery } from '@zendeskgarden/react-theming';
+import { Field, Label, Input, InputGroup } from '@zendeskgarden/react-forms';
+import { Button } from '@zendeskgarden/react-buttons';
+
+const StyledCol = styled(Col)`
+  ${p => mediaQuery('down', 'xs', p.theme)} {
+    margin-top: ${p => p.theme.space.sm};
+  }
+`;
+
+const Example = () => {
+  const [value, setValue] = useState('Soil');
+
+  const onChange = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  }, []);
```

Comment:
Why is `onChange` being demonstrated in this "Size" example?

Reply:
To share the values between the two inputs. This was what we used to do in the stylguidist examples (pre-knobs). Will just remove.